test/Orchard.Tests/DisplayManagement/Decriptors/DefaultShapeTableManagerTests.cs

```diff
@@ -126,7 +125,7 @@ public TestThemeExtensionInfo(string name, IFeatureInfo baseTheme)
                     {"name", name},
                     {"desciption", name},
                     {"type", "theme"},
-                    {"basetheme", baseTheme.Id}
+                    {"basethemes", baseTheme.Id }
```

Comment:
Because an extension supports multiple dependencies, a theme is just an extension, and base theme is a implicit dependency referencer. So we can support multiple base themes.